airflow/example_dags/example_bigquery_to_bigquery.py

```diff
@@ -0,0 +1,34 @@
+import airflow
+from airflow.models import DAG
+from airflow.operators import bash_operator
+from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
+
+args = {
+    'owner': 'Airflow',
+    'start_date': airflow.utils.dates.days_ago(2),
+}
+
+dag = DAG(
```

Comment:
Can you use a context manager?

Reply:
In examples, we prefer to use a context manager.

Reply:
Sure